LEAF_Request_Portal/templates/subindicators.tpl

```diff
@@ -690,6 +690,36 @@
             <input type="text" id="<!--{$indicator.indicatorID|strip_tags}-->" name="<!--{$indicator.indicatorID|strip_tags}-->" value="<!--{$indicator.value|sanitize}-->" style="display: none" />
             <!--{$indicator.html}-->
         <!--{/if}-->
+        <!--{if $indicator.is_sensitive == 1}-->
+        <script>
+        var visible = 0;
+        $('#<!--{$indicator.indicatorID|strip_tags}-->').attr('type', 'password');
+        if($('#<!--{$indicator.indicatorID|strip_tags}-->').is('textarea')) {
+            $('#<!--{$indicator.indicatorID|strip_tags}-->').css('display', 'none');
+        }
+        function toggleSensitive() {
+            if(visible === 0) {
+                $(event.target).css('background-image', 'url(/libs/dynicons/?img=eye_visible.svg&w=16)');
+                if($('#<!--{$indicator.indicatorID|strip_tags}-->').is('textarea')) {
+                    $('#<!--{$indicator.indicatorID|strip_tags}-->').css('display', 'inline');
+                } else {
+                    $('#<!--{$indicator.indicatorID|strip_tags}-->').attr('type', 'text');
+                }
+                $('#sensitiveStatus').attr('aria-label', 'sensitive data shown');
+                visible = 1;
+            } else {
+                $(event.target).css('background-image', 'url(/libs/dynicons/?img=eye_invisible.svg&w=16)');
+                if($('#<!--{$indicator.indicatorID|strip_tags}-->').is('textarea')) {
+                    $('#<!--{$indicator.indicatorID|strip_tags}-->').css('display', 'none');
+                } else {
+                    $('#<!--{$indicator.indicatorID|strip_tags}-->').attr('type', 'password');
+                }
+                $('#sensitiveStatus').attr('aria-label', 'sensitive data hidden');
+                visible = 0;
+            }
+        }
+        </script>
+        <!--{/if}-->
```

Comment:
It works fine if it's just the parent indicator but it goes nuts if there are any children marked as sensitive.

Reply:
Addressed in commit `7a4`